WARNING:absl:You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. 
c:\Users\Pedro\Desktop\LotoIA\Lototurf\LOTOTURF.py:44: RuntimeWarning: invalid value encountered in cast
  prediccion_ajustada = np.round(prediccion).astype(int)
C:\Users\Pedro\AppData\Local\Programs\Python\Python312\Lib\site-packages\sklearn\base.py:493: UserWarning: X does not have valid feature names, but MinMaxScaler was fitted with feature names
  warnings.warn(
c:\Users\Pedro\Desktop\LotoIA\Lototurf\LOTOTURF.py:56: RuntimeWarning: invalid value encountered in cast
  new_predictions = np.rint(new_predictions).astype(int)
Traceback (most recent call last):
  File "c:\Users\Pedro\Desktop\LotoIA\Lototurf\LOTOTURF.py", line 65, in <module>
    df.to_csv('ruta/al/archivo.csv', index=False, encoding='utf-8')
  File "C:\Users\Pedro\AppData\Local\Programs\Python\Python312\Lib\site-packages\pandas\util\_decorators.py", line 333, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Pedro\AppData\Local\Programs\Python\Python312\Lib\site-packages\pandas\core\generic.py", line 3967, in to_csv
    return DataFrameRenderer(formatter).to_csv(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Pedro\AppData\Local\Programs\Python\Python312\Lib\site-packages\pandas\io\formats\format.py", line 1014, in to_csv
    csv_formatter.save()
  File "C:\Users\Pedro\AppData\Local\Programs\Python\Python312\Lib\site-packages\pandas\io\formats\csvs.py", line 251, in save
    with get_handle(
         ^^^^^^^^^^^
  File "C:\Users\Pedro\AppData\Local\Programs\Python\Python312\Lib\site-packages\pandas\io\common.py", line 749, in get_handle
    check_parent_directory(str(handle))
  File "C:\Users\Pedro\AppData\Local\Programs\Python\Python312\Lib\site-packages\pandas\io\common.py", line 616, in check_parent_directory
    raise OSError(rf"Cannot save file into a non-existent directory: '{parent}'")
OSError: Cannot save file into a non-existent directory: 'ruta\al'
